kornia/feature/steerers.py

```diff
@@ -0,0 +1,66 @@
+from typing import Dict
+
+import torch
+
+from kornia.core import Module, Tensor
+from kornia.utils.helpers import map_location_to_cpu
+
+dedode_steerer_urls: Dict[str, str] = {
+    "B-C4": "https://github.com/georg-bn/rotation-steerers/releases/download/release-2/B_C4_Perm_steerer_setting_C.pth",
```

Comment:
maybe we should push to kornia hugging face hub these weights? cc @edgarriba

Reply:
I realized, that for the steerers included currently, we can just define them as they are not trained. So I refactored the code like that instead and removed this dict.